frontend/src/components/CreateBoard/RegularBoard/SettingsTabs/index.tsx

```diff
@@ -19,6 +20,10 @@ const SettingsTabs = () => {
   const haveError = useRecoilValue(createBoardError);
   const setToastState = useSetRecoilState(toastState);
 
+  const handleChangeTab = (value: number) => {
+    setCurrentTab(value);
+  };
```

Comment:
```suggestion
  const handleChangeTab = (value: number) => {
  if (!haveError) return 
  setCurrentTab(value);
  };
```

if this doesn't make sense to you, feel free to say so :D